specification/network/resource-manager/Microsoft.Network/stable/2017-06-01/applicationGateway.json

```diff
@@ -1910,6 +1910,8 @@
       "type": "string",
       "description": "Ssl cipher suites enums.",
       "enum": [
+        "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384",
```

Comment:
I know, it's modeled as a string but still it would be better if we add new enum values at the end.

Reply:
@sergey-shandar done